site/gatsby-site/gatsby-browser.js

```diff
@@ -13,29 +13,26 @@ import 'bootstrap-daterangepicker/daterangepicker.css';
 
 import '@fortawesome/fontawesome-svg-core/styles.css';
 
-const ROUTES_WITH_RULES = ['/cite/', '/apps/discover'];
-
 export const shouldUpdateScroll = ({ routerProps: { location } }) => {
   const { pathname, hash } = location;
 
-  if (ROUTES_WITH_RULES.includes(pathname)) {
```

Comment:
To clarify why I'm changing this, this line never worked.

Inside citation pages, `pathname` equals something like `/cite/10` and `ROUTES_WITH_RULES.includes(pathname)` will never match that because it is doing array level matching not string matching.